setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
+max-line-length = 120
```

Comment:
You probably wouldn't lose the benefits of short lines with 120 characters
Still, we should stick to 80 (what if I want to code on a potato? Also it's pretty much the industry standard)